tests/c_languages/CPPCheckBearTest.py

```diff
@@ -26,10 +27,12 @@
     return 0;
 }"""
 
+test_dir = os.path.join(os.path.dirname(__file__), 'test_files')
 
 CPPCheckBearTest1 = verify_local_bear(CPPCheckBear,
                                       valid_files=(good_file, warn_file),
-                                      invalid_files=(bad_file,))
+                                      invalid_files=(bad_file,),
+                                      settings={'include_paths': test_dir})
```

Comment:
this test worked before ... and works after you add this argument.  Adding this argument here does not test that the argument is used ; it only tests that it doesnt break something.

We need a real test case where `include_paths` is *necessary*, and if omitted the test case would fail.